<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>db_browser</class>
 <widget class="QWidget" name="db_browser">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>516</width>
    <height>470</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Database Browser</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <property name="fieldGrowthPolicy">
    <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
   </property>
   <item row="1" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Select a mac address:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QTableView" name="tableView">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>1</verstretch>
      </sizepolicy>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::DoubleClicked</set>
     </property>
     <property name="dragDropOverwriteMode">
      <bool>false</bool>
     </property>
     <property name="dragDropMode">
      <enum>QAbstractItemView::NoDragDrop</enum>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="showGrid">
      <bool>true</bool>
     </property>
     <property name="gridStyle">
      <enum>Qt::DotLine</enum>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
     <attribute name="horizontalHeaderVisible">
      <bool>true</bool>
     </attribute>
     <attribute name="horizontalHeaderCascadingSectionResizes">
      <bool>false</bool>
     </attribute>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Label:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLineEdit" name="label_in"/>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Address:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QLineEdit" name="mac_in">
     <property name="maxLength">
      <number>17</number>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0">
     <item>
      <widget class="QPushButton" name="save_btn">
       <property name="text">
        <string>Save Changes to Database</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="undo_btn">
       <property name="text">
        <string>Undo Changes</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="5" column="1">
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QPushButton" name="add_btn">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>1</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Add</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="edit_btn">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>1</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Edit</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="del_btn">
       <property name="text">
        <string>Delete</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="6" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>13</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="7" column="0">
    <widget class="QPushButton" name="cancel_btn">
     <property name="text">
      <string>Close</string>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="QPushButton" name="sel_btn">
     <property name="font">
      <font>
       <pointsize>15</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="toolTip">
      <string>Select the MAC address to which you want to send a Magic Packet</string>
     </property>
     <property name="text">
      <string>Select</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>label_in</tabstop>
  <tabstop>mac_in</tabstop>
  <tabstop>add_btn</tabstop>
  <tabstop>edit_btn</tabstop>
  <tabstop>del_btn</tabstop>
  <tabstop>sel_btn</tabstop>
  <tabstop>save_btn</tabstop>
  <tabstop>undo_btn</tabstop>
  <tabstop>cancel_btn</tabstop>
  <tabstop>tableView</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>del_btn</sender>
   <signal>clicked()</signal>
   <receiver>db_browser</receiver>
   <slot>delete_row()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>280</x>
     <y>473</y>
    </hint>
    <hint type="destinationlabel">
     <x>65</x>
     <y>410</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>add_btn</sender>
   <signal>clicked()</signal>
   <receiver>db_browser</receiver>
   <slot>insert_row()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>213</x>
     <y>407</y>
    </hint>
    <hint type="destinationlabel">
     <x>72</x>
     <y>338</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>edit_btn</sender>
   <signal>clicked()</signal>
   <receiver>db_browser</receiver>
   <slot>edit_row()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>195</x>
     <y>438</y>
    </hint>
    <hint type="destinationlabel">
     <x>67</x>
     <y>380</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancel_btn</sender>
   <signal>clicked()</signal>
   <receiver>db_browser</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>171</x>
     <y>643</y>
    </hint>
    <hint type="destinationlabel">
     <x>112</x>
     <y>650</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sel_btn</sender>
   <signal>clicked()</signal>
   <receiver>db_browser</receiver>
   <slot>select_mac()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>285</x>
     <y>609</y>
    </hint>
    <hint type="destinationlabel">
     <x>98</x>
     <y>613</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>delete_row()</slot>
  <slot>insert_row()</slot>
  <slot>edit_row()</slot>
  <slot>select_mac()</slot>
 </slots>
</ui>
